examples/embeddable_examples/public/book/book_embeddable.tsx

```diff
@@ -103,7 +104,12 @@ export class BookEmbeddable extends Embeddable<BookEmbeddableInput, BookEmbeddab
   };
 
   getInputAsValueType = async (): Promise<BookByValueInput> => {
-    return this.attributeService.getInputAsValueType(this.input);
+    const input =
```

Comment:
can we simplify this statement a bit?

Reply:
I realized that getting the `explicitInput` of an embeddable will be a common enough task.
The reason this is needed is so that the 'unlink' or 'add' actions don't make inherited input into explicit input.

I added a helper to the attribute service in the add to library PR https://github.com/elastic/kibana/pull/75098.: https://github.com/elastic/kibana/blob/b41eae8ec032320c742c86c9c3cd567b9f4d98e9/src/plugins/dashboard/public/attribute_service/attribute_service.tsx#L126-L133

 It can be simplified a little bit with optional chaining, and I will update it to reflect that over there.